babel/numbers.py

```diff
@@ -184,9 +191,9 @@ def get_currency_unit_pattern(currency, count=None, locale=LC_NUMERIC):
     return loc._data['currency_unit_patterns']['other']
 
 
-def get_territory_currencies(territory, start_date=None, end_date=None,
-                             tender=True, non_tender=False,
-                             include_details=False):
+def get_territory_currencies(territory: str, start_date: date_ | None = None, end_date: date_ | None = None,
+                             tender: bool = True, non_tender: bool = False,
+                             include_details: bool = False) -> list[str]:
```

Comment:
```suggestion
def get_territory_currencies(
    territory: str,
    start_date: date_ | None = None,
    end_date: date_ | None = None,
    tender: bool = True,
    non_tender: bool = False,
    include_details: bool = False,
) -> list[str]:
```

Reply:
This doesn't seem to take into account `include_details` changing the rtype (which is dumb API design in the first place but here we are... :) )